lib/extras/dec/apng.cc

```diff
@@ -257,6 +257,17 @@ Status DecodeChrmChunk(Bytes payload, JxlColorEncoding* color_encoding) {
   return true;
 }
 
+/** Extract information from 'cLLi' chunk. */
+Status DecodeClliChunk(Bytes payload, float* max_content_light_level) {
```

Comment:
Perhaps the function could return a `StatusOr<float>`, and then line 1136 would be something like:

```c++
JXL_ASSIGN_OR_RETURN(ppf->info.intensity_target, DecodeClliChunk(payload));
```

Reply:
I'm not sure about this, since other chunks like `gAMA` and `cHRM` are also parsed with `JXL_RETURN_IF_ERROR`, so the current state seems better for code consistency.

Reply:
I think this is mainly for historical reasons (we haven’t always had `StatusOr`), but we can also change it later.